client/src/components/common/boost/BoostCalibration.tsx

```diff
@@ -0,0 +1,98 @@
+import React , {useState, useCallback} from 'react';
+import {Button, Card, Form, Col} from 'react-bootstrap';
+
+export interface BoostCalibrationProps {
+    onSet: (calibValue: number) => void;
+    onReset: () => void;
+    distTravelled: number;
+  }
+
+/**
+ * Boost calibration card, displays the calibrated distance and the 
+ * distance travelled by the bike while also allowing user to set the 
+ * calibrated distance
+ * 
+ * @returns {React.Component} Component
+ */
+export default function BoostCalibration({
+    onSet, onReset, distTravelled
+}: BoostCalibrationProps) {
+    // Controls the feedback form
+    const [validated, setValidation] = useState(false);
+    
+    const [calibValue, setCalibValue] = useState(0);
+    // Holds the difference between the current distance and the calibrated distance
+    const [calibDiff, setCalibDiff] = useState(0);
+
+    const handleCalibrationChange = useCallback(
+        (event) => {
```

Comment:
Here `event` has `any` type which typescript seems to be okay about but isn't ideal. Can you add types for `event` here and the next few instances? The types are
`React.ChangeEvent<HTMLInputElement>`
`React.FormEvent<HTMLInputElement>`
`React.KeyboardEvent<HTMLInputElement>`